civihr_employee_portal/views/templates/views-view-table--Tasks--block.tpl.php

```diff
@@ -31,64 +31,76 @@ function taskAssignedToUser($task, $user_id) {
   return strip_tags($task['task_contacts_1']) != $user_id;
 }
 
-$civiUser = get_civihr_uf_match_data($user->uid);
+/**
+ * Wraps the original function that fetches the filter type by task date so that
+ * the resulting filter, if necessary, can be normalized to match the list of filters
+ * offered in the UI ('tomorrow' and 'week' are grouped together)
+ *
+ * @param  [type] $filter [description]
+ * @return [type]         [description]
+ */
+function taskFilterByDate($taskDate, $normalize = false) {
```

Comment:
The best way would be to create this function in php module file (or even modify '_get_task_filter_by_date' to accept this additional $normalize attribute). It's good to not define functions in template files in case that the template would be used twice on the same page (which would cause PHP error regarding double declaration of the same function).